src/Entity/FailedLoginAttempt.php

```diff
@@ -45,13 +45,20 @@ public function __construct(string $signature, array $extra)
 
     public static function createFromRequest(Request $request): self
     {
+        $attempt = LoginAttemptSignature::createFromRequest($request);
+
         return new self(
-            (new FailedLoginAttemptSignature($request->get('_adherent_email'), $request->getClientIp()))(),
+            $attempt(),
```

Comment:
We're loosing some meaning with this invocation, not a fan.

Reply:
Yeap, replaced with `getSignature` method